### PR TITLE
5287: Delete an Order

### PR DESCRIPTION
**Pull request by talawson05**
[AB#8817](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/8817)
Still need data-test-id locators, before enabling tests

### CODE REVIEW

File: src/OrderFormAcceptanceTests.Actions/Pages/DeleteOrder.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+﻿using OpenQA.Selenium;
+using OrderFormAcceptanceTests.Actions.Utils;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OrderFormAcceptanceTests.Actions.Pages
+{
+    public class DeleteOrder : PageAction
```
Comment (sam-roberts-bjss) [inline src/OrderFormAcceptanceTests.Actions/Pages/DeleteOrder.cs:9]: Could add the sealed keyword here to provide context on whether or not inheritance has been considered for this class?

File: src/OrderFormAcceptanceTests.Steps/Steps/DeleteOrder.cs [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using TechTalk.SpecFlow;
+
+namespace OrderFormAcceptanceTests.Steps.Steps
+{
+    [Binding]
+    class DeleteOrder : TestBase
```
Comment (sam-roberts-bjss) [inline src/OrderFormAcceptanceTests.Steps/Steps/DeleteOrder.cs:14]: public? sealed?

File: src/OrderFormAcceptanceTests.Tests/Features/DeleteOrder.feature [hunk 3]
```diff
@@ -0,0 +1,36 @@
+﻿Feature: Delete	Order
```
Comment (sam-roberts-bjss) [inline src/OrderFormAcceptanceTests.Tests/Features/DeleteOrder.feature:1]: Spacing?

File: src/OrderFormAcceptanceTests.Tests/Features/DeleteOrder.feature [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+Scenario: Delete Order - Not chosen to delete order
+	Given the confirm delete page is displayed
+	When the User chooses not to delete the Order
+	Then the Order dashboard is presented
+	And the status of the Order does not change to deleted
+@ignore
+Scenario: Delete Order - Go back to all orders after deleting
+	Given the Order deleted page is presented
+	When the User chooses to go back
+	Then the Organisation Orders Dashboard is displayed
```
Comment (sam-roberts-bjss) [inline src/OrderFormAcceptanceTests.Tests/Features/DeleteOrder.feature:36]: Add a new line to make Github happy?

Comment (talawson05) [reply src/OrderFormAcceptanceTests.Tests/Features/DeleteOrder.feature:36]: under the Then?

Comment (sam-roberts-bjss) [reply src/OrderFormAcceptanceTests.Tests/Features/DeleteOrder.feature:36]: Yes please

**Pull request merged**